### ISSUE TITLE
Cannot enable or disable Per-Member Quotas and Per-Member Shapers

### ISSUE THREAD

**Issue by junelsolis** [ntop/ntopng#1784]
I am running Ntopng Enterprise Edition v.3.4.180517. I tried to make some modifications to the traffic policy today and I found out that whenever I try to disable/enable per-member quotas and/or per-member shapers to a host pool and click 'Save' the page refreshes and reverts back to the original setting.

**Comment by emanuele-f** [comment]
Hi, do you have any errors into the ntopng log? If you are running ntopng as a service, you can get the log with `journactl -u ntopng`. Please note that after some minutes the page cookie times out and any changes applied afterwards the timeout will be reverted back.

**Comment by junelsolis** [comment]
Hello. I am running it as a service. There is nothing in the logs that indicates a problem. I also tried logging in with another browser, attempted to make the changes, and the same issue. Also just discovered that when I go to Traffic Policy-->Manage Policies-->Select Pool Name-->Default Policy, I can set the Traffic Policy to 'Drop All' Traffic-->Save, when I select another pool and then go back to the pool I modified, the original settings are there again, unchanged.
